apps/ledger-live-desktop/src/renderer/components/Web3AppWebview/helpers.ts

```diff
@@ -0,0 +1,21 @@
+import React, { MutableRefObject } from "react";
+
+export function useCombinedRefs<T>(
+  ...refs: (((instance: T | null) => void) | MutableRefObject<T | null> | null)[]
+) {
+  const targetRef = React.useRef<T>(null);
+
+  React.useEffect(() => {
+    refs.forEach(ref => {
+      if (!ref) return;
+
+      if (typeof ref === "function") {
+        ref(targetRef.current);
+      } else {
+        ref.current = targetRef.current;
+      }
+    });
+  }, [refs]);
```

Comment:
`refs` will always change here if I'm not mistaken and re-run the effect on every render, if you still want this solution there is another usage here https://mtsknn.fi/blog/react-fallback-ref/#merge-refs